### ISSUE TITLE
"Dependency Cycle Found" does not give good information on what the loop is

### ISSUE THREAD

**Issue by waddlesplash** [jriecken/dependency-graph#7]
It would be nice if a complete copy of the dependency graph was listed that caused the error (e.g. if `a` indirectly depends on `a` then this might be `a -> c -> d -> b -> a`).

**Comment by jriecken** [comment]
The error message will now include the cycle in it:

```
    var graph = new DepGraph();

    graph.addNode('a');
    graph.addNode('b');
    graph.addNode('c');
    graph.addNode('d');

    graph.addDependency('a', 'b');
    graph.addDependency('b', 'c');
    graph.addDependency('c', 'a');
    graph.addDependency('d', 'a');

    graph.overallOrder(); // throws Error('Dependency Cycle Found: d -> a -> b -> c -> a')
```

**Comment by jriecken** [comment]
Released version 0.4.0

**Comment by waddlesplash** [comment]
Thanks for the fast update!
